deps: Bump jib-core to 0.20.0

manusa opened the pull request:
## Description

Fix #832 

deps: Bump jib-core to 0.20.0

## Type of change
<!---
What types of changes does your code introduce? Put an `x` in all the boxes that apply
-->
 - [ ] Bug fix (non-breaking change which fixes an issue)
 - [ ] Feature (non-breaking change which adds functionality)
 - [ ] Breaking change (fix or feature that would cause existing functionality to change
 - [x] Chore (non-breaking change which doesn't affect codebase;
   test, version modification, documentation, etc.)

## Checklist
 - [x] I have read the [contributing guidelines](https://www.eclipse.org/jkube/contributing)
 - [x] I signed-off my commit with a user that has signed the [Eclipse Contributor Agreement](https://www.eclipse.org/legal/ECA.php)
 - [x] I Added [CHANGELOG](../CHANGELOG.md) entry
 - [ ] I have implemented unit tests to cover my changes
 - [ ] I have updated the [documentation](../kubernetes-maven-plugin/doc) accordingly
 - [ ] No new bugs, code smells, etc. in [SonarCloud](https://sonarcloud.io/dashboard?id=jkubeio_jkube) report
 - [ ] I tested my code in Kubernetes
 - [ ] I tested my code in OpenShift

<!--
Integration tests (https://github.com/jkubeio/jkube-integration-tests)
Please check integration tests and provide/improve tests if necessary.

Open your PR in Draft mode and verify all of the applicable Checklist items before marking your issue as ready
-->

## CQs
- [x] https://dev.eclipse.org/ipzilla/show_bug.cgi?id=23723
- [x] https://dev.eclipse.org/ipzilla/show_bug.cgi?id=23724
- [x] https://dev.eclipse.org/ipzilla/show_bug.cgi?id=23725
- [x] https://dev.eclipse.org/ipzilla/show_bug.cgi?id=23726
- [x] https://dev.eclipse.org/ipzilla/show_bug.cgi?id=23727

manusa commented:
<!-- Eclipse JKube CI bot report -->

rohanKanojia commented:
Build seems to be failing. 

I remember when I had tried upgrading jib-core, I had added exclusions like you too but it didn't work. I was blocked on updating jackson deps in Kubernetes Client https://github.com/fabric8io/kubernetes-client/pull/3428

manusa commented:
> Build seems to be failing.
> 
> I remember when I had tried upgrading jib-core, I had added exclusions like you too but it didn't work. I was blocked on updating jackson deps in Kubernetes Client [fabric8io/kubernetes-client#3428](https://github.com/fabric8io/kubernetes-client/pull/3428)

It fails for other reasons. All Jackson dependencies are loaded from the local dependency management version (now updated too).
Build should work after my last rebase.
